src/main/frontend/db/query_dsl.cljs

```diff
@@ -123,13 +110,8 @@
                  t/days)]
         (tc/to-long (t/plus (t/today) (tf duration)))))))
 
-(defn uniq-symbol
```

Comment:
We no longer need to carry counter state or use this since rules already give us unique binding